File: Beecrowd/Beginner/1101 - 1190/1185 - Above the Secundary Diagonal/C/1185_AbovetheSecundaryDiagonal.c
#include <stdio.h>

int main() {
    char operation;
    double matrix[12][12], sum = 0;

    scanf("%c", &operation);

    for (int i = 0; i < 12; i++) {
        for (int j = 0; j < 12; j++) {
            scanf("%lf", &matrix[i][j]);

            if (j < 11 - i) {
                sum += matrix[i][j];
            }
        }
    }

    if (operation == 'S') printf("%.1lf\n", sum);
    else printf("%.1lf\n", sum / 66.0);

    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1185 - Above the Secundary Diagonal/Python/1185_AbovetheSecundaryDiagonal.py
sum = 0

operation = input()

for i in range(12):
    for j in range(12):
        number = float(input())

        if j < 11 - i:
            sum += number

if operation == 'S':
    print(f"{sum:.1f}")
else:
    print(f"{sum / 66.0:.1f}")
